test/sharness/t0130-multinode.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+#
+# Copyright (c) 2015 Jeromy Johnson
+# MIT Licensed; see the LICENSE file in this repository.
+#
+
+test_description="Test multiple ipfs nodes"
+
+. lib/test-lib.sh
+
+IPTB_ROOT=`pwd`/testbed
+
+test_expect_success "set up a few nodes" '
+	iptb -n=3 init &&
+	iptb -wait start
+'
+
+test_expect_success "add a file on node1" '
+	IPFS_PATH="$IPTB_ROOT/1"
```

Comment:
hrm, this test runs fine (and correctly) for me. Can change to `&&` if you want though, no big deal